library/user.inc

```diff
@@ -136,3 +136,34 @@ function prevSetting($uspfx, $postvar, $label, $initval)
 
     return $pset;
 }
+
+/**
+ * Function to set the state of expandable forms as per user choice, user default or global default
+ * @param array $filenames - array containing the names of the files being targeted, first element contains name of current file
+ * @param string $current_filename - derived from the first element of array $filenames
+ * @param integer $global_value - either the user defined or administrator defined global value
+ * @param integer $current_state - the current state of the form either 0 or 1
```

Comment:
should remove the params that have been removed from above comments

Reply:
this still needs to be done:
should remove the params that have been removed from above comments